benchmark.py

```diff
@@ -0,0 +1,57 @@
+import argparse
+import os
+import subprocess
+
+import utils
+
+def KillBrowsers(browser_list):
+  for browser in browser_list:
+    browser_executable = utils.browsers_definition[browser]['executable']
+    subprocess.call(["killall", "-9", browser_executable])
+
+def Record(scenario_name, driver_script, output_dir, browser=None, extra_args=[], background_script=None):
+
+  with open("powermetrics.plist", "w") as f:
+    powermetrics_process = subprocess.Popen(["sudo", "powermetrics", "-f", "plist", "--samplers", "all", "--show-responsible-pid", "--show-process-gpu", "--show-process-energy", "-i", "60000"], stdout=f, stdin=subprocess.PIPE)
```

Comment:
I would really prefer if we could start power_metrics only once.

Is it not possible to do the "tail" trick anymore because were outputting plist?

Reply:
Of course I don't mean literally running tail. We could use whatever is available in Python.

Reply:
As discussed offline this is too unstable for now. Let's push as is and reevaluate.